src/libstd/sys/sgx/abi/entry.S

```diff
@@ -121,6 +121,15 @@ sgx_entry:
     fnstcw %gs:tcsls_user_fcw
 /*  reset user state */
     cld /* x86-64 ABI requires DF to be unset at function entry/exit */
+
+/*  making sure AC flag is not set in rflags */
+/*  avoid using the 'clac' instruction to be compatible with older compilers */
+    pushfq
+    popq %rcx
+    and $0xFFFFFFFFFFFBFFFF, %rcx
+    push %rcx
```

Comment:
You can clear DF and AC in a single instruction. e.g. `andq $~0x40400, (%rsp)`

Reply:
agreed. Fixed :)